app.js

```diff
@@ -88,7 +88,8 @@ app.use(helmet.contentSecurityPolicy({
   directives: {
     defaultSrc: [
       'www.youtube.com',
-      'https://public.etherpad-mozilla.org'
+      'https://public.etherpad-mozilla.org',
+      'https://pontoon.mozilla.org'
```

Comment:
Why'd you add it to every directive?

Surely pontoon doesn't need to be allowed by default.